Makefile

```diff
@@ -40,7 +40,8 @@ azd-login: ## 🔑 Login to Azure with azd and a SPN
 
 deploy: azd-login ## 🚀 Deploy everything to Azure
 	@echo -e "\e[34m$@\e[0m" || true
-	@azd env set AZURE_APP_SERVICE_HOSTING_MODEL code
+	@azd env new ${AZURE_ENV_NAME}
```

Comment:
AZURE_ENV_NAME is a GitHub setting - now we are not persisting the environment - maybe we could use a commit number so we dont end up being so dependent on the current value. (Not for this PR!)